src/main/java/springboot/onlinebookstore/validation/status/Status.java

```diff
@@ -0,0 +1,20 @@
+package springboot.onlinebookstore.validation.status;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Documented
+@Constraint(validatedBy = StatusValidator.class)
+@Target({ElementType.TYPE, ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Status {
```

Comment:
delete this annotation.
We should not create unique annotation for each enum in the app